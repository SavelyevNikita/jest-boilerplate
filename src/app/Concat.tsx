
import * as React from 'react';

interface ConcatProps {
    str: string;
}

export const Concat =
(props: ConcatProps): React.FunctionComponentElement<ConcatProps> => {
    return (
        <div>{props.str}</div>
    );
}
