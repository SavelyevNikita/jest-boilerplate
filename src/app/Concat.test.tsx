
import * as React from 'react';
import { shallow } from 'enzyme';
import { Concat } from './Concat';

test('Concat exists', (): void => {
    const el = (<Concat str=""></Concat>);
    const sh = shallow(el);
    expect(sh).toMatchSnapshot();
});

test('Concat accepts one parameter', (): void => {
    const str = '2';
    const el = (<Concat str={str}></Concat>);
    const sh = shallow(el);
    expect(sh.text()).toBe(str);
});
