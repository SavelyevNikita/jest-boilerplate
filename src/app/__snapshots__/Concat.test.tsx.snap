// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Concat exists 1`] = `<div />`;
